text/0265-event-vision.md

```diff
@@ -0,0 +1,72 @@
+# Event basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to write down and agree on a few basics.
+
+### What are events?
+
+Events are a type of log which has a required event name and a specific structure which is implied by that event name.
+
+The event name and its implied structure make events much more suitable for observability compared to traditional logs.
+
+Because of this, the OpenTelemetry project wants to encourage the transition from traditional logs to events.
```

Comment:
wants to encourage the transition from unstructured logs to structured logs, and events is one way to achieve this goal.

^ is this the intent?

Reply:
Events are not just structured logs, they're structured logs with semantic rigor. The name field is required, providing a reliable invariant for classifying the type of thing that happened.  The fields of the body are (often) governed by semantic conventions, such that for a given event name, consumers can rely on the schema.